packages/StatusTracker/src/components/Point/Point.js

```diff
@@ -106,13 +107,33 @@ const Point = React.forwardRef((props, ref) => {
       );
     }
 
+    if (kind === kinds.CURRENT) {
+      return (
+        <sc.Point
+          aria-label={name}
+          onMouseOver={handleTooltipOpen}
+          onMouseOut={handleTooltipClose}
+          onFocus={handleTooltipOpen}
+          onBlur={handleTooltipClose}
+          isDropdown={false}
+          kind={kind}
+          tabIndex={0}
+          data-pka-anchor="status-tracker.point"
+          {...a11yAttributes}
+        >
+          {name}
+        </sc.Point>
+      );
+    }
+
```

Comment:
Hmmm, instead of having two `return` statements, do you think it's better to have just one?

```
isCurrent = kind === kinds.CURRENT
name = isCurrent ? name : " ";
as = isCurrent ? null : RawButton;
tabIndex = isCurrent ? 0 : null;

return (
  <sc.Point
    ...
  />
);

Reply:
There's are 3 in total 😄 personally I prefer to show all the props within the returned element to make it easier to understand